app/views/layouts/customer_layout.html.erb

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Validations</title>
```

Comment:
The titles are what the user sees when they hover over a tab. It is good for them to be descriptive in case you have a lot of tabs open and forget which is which. The title helps uses find the right tab